draft-ietf-dnsop-svcb-https.md

```diff
@@ -1702,6 +1702,259 @@ the authoritative.  Thus, splitting the RR type would double, or in
+backslash in two presentation formats.
+
+    16 foo.example.org alpn="f\\\\oo\\,bar,h2"
+    16 foo.example.org alpn=f\\\092oo\092,bar,h2
+
+    \# 35 (
+    00 10                                              ; priority
+    03 66 6f 6f 07 65 78 61 6d 70 6c 65 03 6f 72 67 00 ; target
+    00 01                                              ; key 1
+    00 0c                                              ; param length 12
+    08                                                 ; alpn length 8
+    66 5c 6f 6f 2c 62 61 72                            ; alpn value
+    02                                                 ; alpn length 2
+    68 32                                              ; alpn value
+    )
+
+    \x00\x10                                           # priority
+    \x03foo\x07example\x03org\x00                      # target
+    \x00\x01                                           # key 1
+    \x00\x0c                                           # param length 12
+    \x08                                               # alpn length 8
+    f\oo,bar                                           # alpn value
+    \x02                                               # alpn length 2
+    h2                                                 # alpn value
+
+
+## Failure cases
+
+In this subsection, example resource records are shown which are not
+compliant with this document. The various reasons for non-compliance
+are explained with each example.
+
+
+This example has mutilple instances of the same
+SvcParamKey {{presentation}}.
+
+    example.com.   SVCB   1 foo.example.com. (
+                           key123=abc key123=def
+                           )
+
+The next example the SvcParams are missing their values.
+
+    example.com.   SVCB   1 foo.example.com. (
+                           alpn port ipv4hint echconfig ipv6hint
+                           )
+
+The "no-default-alpn" SvcParamKey value must be empty ({{alpn-key}}).
+
+    example.com.   SVCB   1 foo.example.com. no-default-alpn=abc
+
+In this record a mandatory SvcParam is missing ({{mandatory}}).
+
+    example.com.   SVCB   1 foo.example.com. mandatory=key123
+
+The "mandatory" SvcParamKey should not be included in mandatory
+list ({{mandatory}}).
+
+    example.com.   SVCB   1 foo.example.com. mandatory=mandatory
+
+Here there are multiple instances of the same SvcParamKey in
+the mandatory list ({{mandatory}}).
+
+    example.com.   SVCB   1 foo.example.com. ( 
+                          mandatory=key123,key123 key123=abc
+                          )
+
+With HTTPS records the SvcParamKeys "port" and "no-default-alpn" are
+automatically mandatory and therefore are required in the
+record ({{value-list}}).
+
+    example.com.   HTTPS   1 foo.example.com. (
+                           mandatory=key123 key123=123
+                           )
+
+
+In the last example, the automatically mandatory SvcParamKeys 
+"port" and "no-default-alpn" should not appear in the 
```

Comment:
As your text notes, this is a "should not", so the record is not invalid.

Reply:
and removed, thanks for the feedback Ben!